syntax = "proto3";

package login_msg;

//---------------------------------------


message MSG_REQUEST_CLIENT_LOGIN
{
	uint64 user_id = 1;
	int32 version = 2;
	bytes session_key = 3;
}

message MSG_RESPONSE_CLIENT_LOGIN
{
	int32 status_code = 1;
	uint64 user_id = 2;
	int32 version = 3;
	bool is_newbie = 4;
}

message MSG_REQUEST_ECHO
{
	uint64 value1 = 1;
	string value2 = 2;
}

message MSG_RESPONSE_ECHO
{
	uint64 value1 = 1;
	string value2 = 2;
}



message MSG_Test1{
	uint64 id1 = 1;
	string test1_str = 2;
}


message MSG_Test2{
	uint64 id2 = 1;
	string test2_str = 2;
}

message MSG_Test3{
	uint64 id3 = 1;
	string test3_str = 2;
}


message MSG_Test4{
	uint64 id4 = 1;
	string test4_str = 2;
}

message MSG_Test5{
	uint64 id5 = 1;
	string test5_str = 2;
}